recipe/patches/0001-use-include-path-of-our-libxml2.patch

```diff
@@ -0,0 +1,16 @@
+On windows, our xmlversion.h is in Library/include/libxml/xmlversion.h, whereas
+on linux, it's in include/libxml2/libxml/xmlversion.h as expected.
+
+Index: ffmpeg/configure
+===================================================================
+--- ffmpeg.orig/configure	2024-06-25 16:19:10.075302986 -0500
++++ ffmpeg/configure	2024-07-10 17:05:14.046340384 -0500
+@@ -6855,7 +6855,7 @@
+ enabled libzvbi           && require_pkg_config libzvbi zvbi-0.2 libzvbi.h vbi_decoder_new &&
+                              { test_cpp_condition libzvbi.h "VBI_VERSION_MAJOR > 0 || VBI_VERSION_MINOR > 2 || VBI_VERSION_MINOR == 2 && VBI_VERSION_MICRO >= 28" ||
+                                enabled gpl || die "ERROR: libzvbi requires version 0.2.28 or --enable-gpl."; }
+-enabled libxml2           && require_pkg_config libxml2 libxml-2.0 libxml2/libxml/xmlversion.h xmlCheckVersion
++enabled libxml2           && require_pkg_config libxml2 libxml-2.0 libxml/xmlversion.h xmlCheckVersion
```

Comment:
I have a feeling that we should fix our libxml2 package. conda-forge correctly have `Library/include/libxml2` and so should we.

Reply:
agreed. https://github.com/AnacondaRecipes/libxml2-feedstock/pull/16

Reply:
well, there's a libxml2 buildout happening at the moment. I need to wait for that so I don't get solver errors. Moving this to blocked so we don't have to do the work twice

Reply:
actually, this is a snowflake-requested package (or at least a dependency for one, torchaudio). Are you ok if I leave the patch in with a note to remove it once the libxml2 buildout is done?

Reply:
I've implemented that

Reply:
Great, thanks. Don't forget to remove the patch.

Reply:
@JeanChristopheMorinPerso heads up to this thread. are you ok with leaving the patch in until the libxml2 buildout is done. Otherwise, this is held up for the customer. I think it's acceptable given that libxml2 is fixed.

Reply:
That's fine. Once we get to the rebuild, it will trip on the updated libxml2 and we'll remove the patch.